test/extended/networking.sh

```diff
@@ -7,9 +7,19 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-if [[ "${OPENSHIFT_QUIET_OUTPUT:-false}" != "true" ]]; then
+CONFIG_ROOT="${OPENSHIFT_CONFIG_ROOT:-}"
+
+# CI mode (deploying clusters for each plugin) should be noisy by
+# default to aid in debugging.
+if [[ "${CONFIG_ROOT}" = "" ]]; then
+  DEFAULT_QUIET=false
+else
+  DEFAULT_QUIET=true
+fi
+
+if [[ "${OPENSHIFT_QUIET_OUTPUT:-${DEFAULT_QUIET}}" != "true" ]]; then
```

Comment:
My other challenge is internalizing that idiomatic bash would appear to require flags intended to be boolean as binary (not present or present) rather than depending on value.  I don't find it intuitive that OPENSHIFT_QUIET_OUTPUT=false be equivalent to OPENSHIFT_QUIET_OUTPUT=true.  That might make sense in bash, but in most non-bash applications a parameter value means something.